packages/react/src/ComparisonTable/ComparisonTable.tsx

```diff
@@ -55,8 +55,7 @@ export const _ComparisonTable = forwardRef(
             children: React.Children.map(child.props.children, (rowChild, rowChildIndex) => {
               if (rowChild.type === Cell) {
                 return React.cloneElement(rowChild, {
-                  as: 'th',
-                  'aria-hidden': !rowChild.props.children ? 'true' : undefined,
+                  as: rowChild.props.children ? 'th' : 'td',
```

Comment:
Is the conditional behaviour needed here? Wouldn't it always be a `th` for the first row?

Reply:
Column alignment is also off for the first row... not sure if related to this or the removal of aria-hidden

[before](https://stunning-chainsaw-j82glqz.pages.github.io/?path=/story/components-comparisontable--minimal)
![Screenshot 2024-08-15 at 17 35 45](https://github.com/user-attachments/assets/885af181-4ff3-49a3-b7de-a77a3cbc42e9)


[after](https://primer-52bb15c8a0-26139705.drafts.github.io/brand/storybook/?path=/story/components-comparisontable--minimal)
![Screenshot 2024-08-15 at 17 35 52](https://github.com/user-attachments/assets/1af737fc-9745-4384-996d-a01c781654d4)

Reply:
Sorry, I should have added context on the `th`/`td` behaviour here (and checked the snapshots after making the change!).

I added this as axe was complaining about empty `th` cells. You can see the rule here

https://dequeuniversity.com/rules/axe/4.7/empty-table-header?application=axeAPI

I couldn't find a recommended approach for this kind of 2-way table where the top-left cell doesn't have any meaning, hence why I landed on the `td` approach.

Thanks for spotting the CSS issues. Fixing the CSS might take a bit of extra work as there are a lot of selectors which expect that first cell to be a `th`.

Let me dig into this a bit more and find an approach that works.

Reply:
Sorry, I should have added context on the `th`/`td` behaviour here (and checked the snapshots after making the change!).

I added this as axe was complaining about empty `th` cells. You can see the rule here

https://dequeuniversity.com/rules/axe/4.7/empty-table-header?application=axeAPI

I couldn't find a recommended approach for this kind of 2-way table where the top-left cell doesn't have any meaning, hence why I landed on the `td` approach.

I've reached out to Tetralogical to see if they have a recommended approach in this scenario

Reply:
Based on [the suggestion from Tetralogical](https://github.com/github/primer/issues/3753#issuecomment-2293478584) I've opted to keep this as a `<td>` and have fixed the style issues